applications/tokenomics-survey-2022.md

```diff
@@ -0,0 +1,327 @@
+- **Contact Email:** mark@taqtiqa.com
+
+### Legal Structure
+
+- **Registered Address:** 9681 41st St NE, Saint Michael, MN 55376
+- **Registered Legal Entity:** Begley Brothers Inc.
+
+### Team's experience
+
+Research interests
+
+- Time series analysis of price series and pricing model performance.
+- Market Microstructure theory and empirical studies.
+- Option pricing and real options/asset pricing.
+
+The University of Western Australia (1994-2003):
+Postgraduate and undergraduate teaching and research supervision. Portfolio Management, Corporate Finance and Derivatives Securities. Co-developed (with Prof R. Maller and Prof A. Szimayer) the University’s joint Undergraduate and Postgraduate Degrees in Mathematics and Finance.
+
+The University of Sydney (2003-2008):
+Postgraduate and undergraduate teaching and research supervision. Advanced Asset Pricing, Corporate Finance, Investments and Advanced Portfolio Management
+
+#### Publications
+
+- [Order imbalance period by period](https://www.researchgate.net/publication/228464696_Order_imbalance_period_by_period), 2008, Asli Ascioglu, Thomas H. Mcinish (Wunderlich Chair) and Mark Van de Vyver
+- [Testing for reduction to random walk in autoregressive conditional heteroskedasticity models](https://doi.org/10.1111/1368-423X.t01-1-00090), December 2002, Econometrics Journal, Claudia Kluppelberg, Ross Maller, Mark Van de Vyver, Derick Wee
+- [How Might Companies Value Esos?](https://doi.org/10.1111/j.1835-2561.2002.tb00191.x), March 2002, Australian Accounting Review, Ross Maller, Rosemary Tan, Mark Van de Vyver
+
+### Team Code Repos
+
+Personal
+
+- https://github.com/taqtiqa-mark
+- https://github.com/bbros-dev
+- https://github.com/taqtiqa
+- https://github.com/BegleyBrothers
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/tatiqa/
+
+## Development Status :open_book:
+
+- links to improvement requests for proposals or [RFPs](https://github.com/w3f/Grants-Program/tree/master/rfp-proposal):
+  - [Designing UpChain: a framework for securing Substrate runtime upgrades and Substrate network upgrades](https://github.com/w3f/Grants-Program/pull/1309)
+- As requested, I have adjusted the source of the research to eliminate myself as the source of bias in the collection of research papers.
+- academic publications relevant to the problem:
+  One of the products from this research will be such a listing under three categories:
+  1. Published articles: As requested, to eliminate bias (and constrain budget) in the collection of research papers, the initial source of the published research will be the top-10 articles provided by the State Library of New South Wales information search service in response to the following query, "Refereed articles on block-chain token-economics using rational expectations equilibrium (a.k.a. no-arbitrage) arguments/analysis, ranked by journal impact factors"
```

Comment:
> I don't think that the ...

As you notice these choices are driven by ... researcher bias (per you're earlier feedback). To my mind what you are calling 'bias' is, in this context, actually skill and experience. However, it was also correctly pointed out that the assessors have no idea about what that skill/experience level is. Hence the direction to bring the budget down.

@Noc2 is the article selection really going to be micro-managed in the proposal stage by a reviewer who, in their own words asked that individual bias be removed where possible:

> Researcher bias is more on how the researcher extracts data ...

I've moved the list of articles to be reviewed into it's own deliverable under Milestone 1.
My understanding is that you can withhold a payment or terminate the grant if the deliverables are as low quality as @dsm-w3f is expecting.

> Another option is to remove academic papers from the scope of the review since you will probably not find anyone regarding polkadot/kusama parachains tokenomics.

This has been addressed:

> This project provides a annotated bibliography to help think through various choices and their trade-offs. It does so by reviewing the current state of play, as it is described in: a) published general-blockchain research, b) Polkadot whitepapers.

> [This one](https://jbba.scholasticahq.com/article/57668.pdf) is worthwhile to take a look.

Thanks for the reference. If that article meets the selection criteria it will be reviewed.

Reply:
@taqtiqa-mark let me explain again my concern about researcher bias quoting the full statement that did before and not only one part of it.

> * Researcher bias is more on how the researcher extracts data and analyses it since could be influenced by its background and beliefs than to set a fixed scope for data for analysis. Usually, research bias is reduced by having more than one researcher doing data analysis in an independent way and comparing the results. As this cannot be done when having only one researcher at least plan to declare this threat to the validity in the paper.

If you want to know more about researcher bias a good reference is [this one](https://doi.org/10.1002/9781119547167.ch76). For knowing more about academic systematic review and the rigor of it I suggest the [seminal work of Kitchenham](https://www.researchgate.net/publication/302924724_Guidelines_for_performing_Systematic_Literature_Reviews_in_Software_Engineering). 

You proposal for trying to solve this bias does not solves the problem and creates another one, as I stated before. Even you commissioned a service to search for literature, there are several other steps in the literature review that will be performed only by yourself, being subject to researcher bias.

> I don't think that the selection of a New South Wales information search service for doing research on published articles is a good idea. This does not reduce bias, in fact, adds a selection bias since the service is probably less complete than other ones. There are few research articles published regarding this specific theme. I used the service cited and did not find any relevant article. I used Google Scholar with the query string ("blockchain" + "token" + "economics" + "rational expectations equilibrium") and also found almost nothing strongly related to the theme (google scholar returned 17 papers in total). I think you can use a more broad search engine for doing this research (like Google Scholar or Scopus) and I bet your scope of review is still small. Another advantage is that you can argue that you have a more strong methodology to try to publish the paper in a good venue. In fact, most of the papers returned in my quick search are more to a related work section than to compose the body of the literature review.

As you [stated in the answer to another question](https://github.com/w3f/Grants-Program/pull/1348#discussion_r1091150144), this is not academic/scientific research anymore. So, this rigor is not needed.

Reply:
Apologies for any misunderstanding, but I am confused that you haven't resolved this comment.
Can you please state what you would like to be done?

Reply:
For me is resolved.